src/extensions/default/bramble/lib/BrambleCodeSnippets.js

```diff
@@ -0,0 +1,34 @@
+/*jslint vars: true, plusplus: true, devel: true, nomen: true, regexp: true, indent: 4, maxerr: 50 */
+/*global define, brackets */
+
+define(function (require, exports, module) {
+    "use strict";
+	
+    var CommandManager = brackets.getModule("command/CommandManager");
+    var EditorManager  = brackets.getModule("editor/EditorManager");
+
+    var CMD_ADD_CODE_SNIPPET_ID     = "bramble.addCodeSnippet";
+	
+    /* Example Snippet */
+    /*var snippet = 
+    '<p> Hello World </p>'*/
+	
+    function init() {
+        /**
+         * Inserts a code snippet into the editor.
+         * @param {string} snippet The snippet to insert.
+         */
+        function addCodeSnippet(snippet) {
+            var editor = EditorManager.getCurrentFullEditor();
```

Comment:
let's change this to `getFocusedEditor` instead of `getCurrentFullEditor` since otherwise, even if your cursor is not in the editor, it will allow you to add a code snippet where your cursor was before it left the editor (which is weird UX).